rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -233,6 +233,11 @@ std::string SqliteStorage::get_storage_identifier() const
   return "sqlite3";
 }
 
+std::string SqliteStorage::get_relative_path() const
```

Comment:
I'd rename this to `get_current_file_path()` or similar.

Reply:
Since `Writer` handles bagfile splitting, the storage plugin only knows about a single file path.